setup.py

```diff
@@ -85,6 +84,7 @@ def get_package_version():
     maintainer='Tom Kralidis',
     maintainer_email='tomkralidis@gmail.com',
     url='https://github.com/geopython/geolinks',
+    install_requires=read('requirements.txt').splitlines(),
```

Comment:
This file is not in the tarball on PyPI making it impossible to build from it source.